### ISSUE TITLE
Login - Broken Aria causes screen reader not to announce tooltip

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#21]
RSAMS Bugs
URL: https://hcdev.openplus.ca/en/user/login
User Agent:  Chrome (Version 92.0.4515.107 (Official Build) (64-bit)); Firefox (90.0.2 (64-bit));
Screen reader tested: NVDA (Ver. 20.0.4) Jaws (Ver. 2020.2008.24 ILM
Language: EN and FR

Issue Description:   When using the keyboard, tooltip information is not being announce by the screen reader.  However, it is announced when using the mouse.  See screenshots below...

Steps to reproduce: 

1. Turn on NVDA or Jaws screen reader
2. Enter login info (hcdev/hc) into pop-up screen to access environment
3. On login page navigate via keyboard (TAB-key) to the username field,
4.  Wait for screen reader to stop speaking (Problem Occurs here)
5. Navigate (TAB-key)  to the password field   
6. Wait for screen reader to stop speaking (Problem Occurs here)

Actual Result: 
Screen reader does not read the tooltip message

Expected results:
The following messages should be read by their respective fields:

- Username: Enter your RSAMS username
- Password: Enter the password that accompanies your username

If you fix the broken-aria, the screen reader should be able to announce it.
![Login-Broken Aria labels](https://user-images.githubusercontent.com/88147013/127677627-9260fb7a-cce1-4cab-9f1d-61b18c993f56.JPG)
![Login-tooltip is not accessible upon field focus-Password](https://user-images.githubusercontent.com/88147013/127677629-de26d16c-0907-41a1-b1c1-02271e7dfbdb.jpg)
![Login-tooltip is not accessible upon field focus-Username](https://user-images.githubusercontent.com/88147013/127677631-a052e9a3-62a4-4f2e-b294-cb7ee21522db.jpg)

**Comment by echouin** [comment]
@hydegeoff OP has confirmed that "Tool tips were disabled before MVP launch as part of initial A11y testing"

This is issue is now considered resolved.

**Comment by hydegeoff** [comment]
Tooltips no longer exist, issue resolved.
